test/index.spec.js

```diff
@@ -102,6 +102,25 @@ describe('desciple-core-api', () => {
 
       expect(exportedData[attestorSsid.did][attestationLink]['agree'][ssid.did][claimlink2]).to.deep.equal({ 'need': 'wine' })
     })
+
+    it('should be able to export multiple (linked) verifiable claims in a channel in order', async () => {
+      let ssid = await discipl.newSsid('memory')
+      let claimlink1 = await discipl.claim(ssid, { 'need': 'beer' })
+      let claimlink2 = await discipl.claim(ssid, { 'need': 'wine' })
+
+      let ssid2 = await discipl.newSsid('memory')
+      let claimlink3 = await discipl.claim(ssid2, { 'need': 'water' })
+
+      let attestationLink = await discipl.attest(ssid, 'agree', claimlink3)
+      let exportedData = await discipl.exportLD(ssid)
+
+      expect(Object.keys(exportedData[ssid.did])[0]).to.equal(claimlink1)
```

Comment:
Still pretty sure that this is not actually guaranteeing order. If the order matters, we should be using an array, if the order doesn't matter (as in, should be inferred from 'previous'-properties), then we probably need a way to signal what's the last one.

Reply:
According to following article js is the only language that actually preserves the order. The JSON spec does not require it so it is language dependent. But i agree ot is more robust not to rely on default behaviour of js engines

https://medium.com/@ty0h/preserving-json-object-keys-order-in-javascript-python-and-go-language-170eaae0de03

Reply:
While the article you're linking refers to parsing json specifically, where we are building an object dynamically, it does turn out that the major browsers do iterate in the order in properties are added, but there are known pitfalls:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys#Description
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Deleted_added_or_modified_properties
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete#Cross-browser_notes

Reply:
ok changed implementation so that it places claims in a channel as an array list of JSON objects